articles/network-watcher/network-watcher-packet-capture-manage-cli.md

```diff
@@ -100,29 +106,21 @@ Once the preceding steps are complete, the packet capture agent is installed on
 
 ### Step 1
 
-The next step is to retrieve the Network Watcher instance. TThe name of the Network Watcher is passed to the `az network watcher show` cmdlet in step 4.
-
-```azurecli
-az network watcher show --resource-group resourceGroup --name networkWatcherName
-```
-
-### Step 2
-
```

Comment:
I believe this may actually be necessary because Step 3 of the current version of the document (where a packet capture is created) contains an inaccuracy. It most likely requires the --location parameter (which is currently missing) and thus finding the Network Watcher instance to use early on is in fact a good idea.

Reply:
Apologies that I'm not understanding this, but there is no `az network watcher show` command in the CLI.  For what's currently Step 1, you said that it's necessary, but by that, do you mean that a user should first run something like `az network watcher configure --resource-group NetworkWatcherRG --locations westcentralus --enabled` to create an instance before proceeding with identifying a storage account and then actually running `az network watcher packet-capture create`?

Reply:
I may be mistaken here (my experience with CLI is limited), but I noticed what seems to be an inconsistency. Specifically:

"az network watcher packet-capture show-status" is documented here as requiring the "--location" parameter to point to a specific Network Watcher resource.

"az network watcher packet-capture create" is documented here as not requiring such a parameter.

This inconsistency makes me believe that the latter is a bug, that is to say the parameter is actually required in both cases (this is certainly so in PowerShell, for example), but the documentation is flawed.

Hence, the proposed change to remove the prior retrieval of the Network Watcher instance may in fact be based on a faulty assumption. If that is the case, then the correct fix is to keep Step 1, and further add the "--location" parameter to the recommended "az network watcher packet-capture create" command line.

Does this make sense?

Reply:
I think this makes sense, though like you mentioned, `az network watcher packet-capture create` does not take --location.  Based on what you suggest for the correct fix, I re-added the original step 1 and added --location for `az network watcher packet-capture create` (assuming that adding --location to the CLI can and will be fixed separately)

Reply:
Oh, I think I understand now. You were right the first time around.

CLI intentionally deduces the location in the  packet-capture create call because it can (it has the VM resource ID). The query call however cannot deduce it because all it sees is the resource name.

Please proceed with your original change here, sorry about the confusion.